### ISSUE TITLE
RIQ Format save issue

### ISSUE THREAD

**Issue by KLONOA9X6** [RHeavenStudio/HeavenStudio#461]
### Describe the bug

When save the old format project to new format, open the new format project again, the following screen will appear
![image](https://github.com/RHeavenStudio/HeavenStudio/assets/95112579/ca6e527c-248b-4dd8-8e74-f36c9c3ff15f)

### How to reproduce the bug?

1. Open the old format project
2. Save as new format project
3. Open the new format project

### What Operating System(s) were you using?

Windows 10/11

### Log file

```shell
......

Imported RIQ successfully!
Detected "v1" riq (RiqBeatmapData)
Running beatmap update handlers
Preprocessing beatmap...
Failed to load remix: Cannot convert type 'Newtonsoft.Json.Linq.JObject' to 'UnityEngine.Vector2'
NullReferenceException: Object reference not set to an instance of an object
  at HeavenStudio.DiscordRPC.DiscordController.UpdateActivity (System.String stateText, System.String stateDetails, System.Boolean updateTime) [0x00000] in <538fe8d04e8e4e23b0a71cd564dc69bf>:0 
  at HeavenStudio.DiscordRPC.DiscordRPC.UpdateActivity (System.String state, System.String details, System.Boolean updateTime) [0x0000a] in <538fe8d04e8e4e23b0a71cd564dc69bf>:0 
  at HeavenStudio.GlobalGameManager.UpdateDiscordStatus (System.String details, System.Boolean editor, System.Boolean updateTime) [0x0001a] in <538fe8d04e8e4e23b0a71cd564dc69bf>:0 
  at HeavenStudio.Editor.Editor.UpdateEditorStatus (System.Boolean updateTime) [0x0000a] in <538fe8d04e8e4e23b0a71cd564dc69bf>:0 
  at HeavenStudio.Editor.Editor.<OpenRemix>b__55_0 (System.String[] paths) [0x0008e] in <538fe8d04e8e4e23b0a71cd564dc69bf>:0 
  at SFB.StandaloneFileBrowserWindows.OpenFilePanelAsync (System.String title, System.String directory, SFB.ExtensionFilter[] extensions, System.Boolean multiselect, System.Action`1[T] cb) [0x0000d] in <6487fc2d5f77467f9734ac6d30f15acb>:0 
  at SFB.StandaloneFileBrowser.OpenFilePanelAsync (System.String title, System.String directory, SFB.ExtensionFilter[] extensions, System.Boolean multiselect, System.Action`1[T] cb) [0x00000] in <6487fc2d5f77467f9734ac6d30f15acb>:0 
  at HeavenStudio.Editor.Editor.OpenRemix () [0x0007c] in <538fe8d04e8e4e23b0a71cd564dc69bf>:0 
  at UnityEngine.Events.InvokableCall.Invoke () [0x00010] in <670c708f396c4df4bd34eacc99554521>:0 
  at UnityEngine.Events.UnityEvent.Invoke () [0x00022] in <670c708f396c4df4bd34eacc99554521>:0 
  at UnityEngine.UI.Button.Press () [0x0001c] in <791940bdae914492aa922d5a2c71ed02>:0 
  at UnityEngine.UI.Button.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) [0x00009] in <791940bdae914492aa922d5a2c71ed02>:0 
  at UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) [0x00007] in <791940bdae914492aa922d5a2c71ed02>:0 
  at UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) [0x00067] in <791940bdae914492aa922d5a2c71ed02>:0 
UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:LogException(Exception)
UnityEngine.EventSystems.ExecuteEvents:Execute(GameObject, BaseEventData, EventFunction`1)
UnityEngine.EventSystems.StandaloneInputModule:ReleaseMouse(PointerEventData, GameObject)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMousePress(MouseButtonEventData)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMouseEvent(Int32)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMouseEvent()
UnityEngine.EventSystems.StandaloneInputModule:Process()
UnityEngine.EventSystems.EventSystem:Update()

......
```

### RIQ file

[INTERNET YAMERO (DANGER!!!).zip](https://github.com/RHeavenStudio/HeavenStudio/files/11713883/INTERNET.YAMERO.DANGER.zip)

### Additional context

Heaven Studio version: git commit: [e909d13](https://github.com/RHeavenStudio/HeavenStudio/commit/e909d13a7a9c48696e457257b4eb525e4ab6e1a9)

**Comment by KLONOA9X6** [comment]
Wait, this issue occurs in any save action.

**Comment by minenice55** [comment]
seems to happen because of https://github.com/RHeavenStudio/HeavenStudio/blob/master/Assets/Scripts/Minigames.cs#L185
it's a read issue in HS userland and not a saving issue in Jukebox luckily

**Comment by minenice55** [comment]
fixed by #463
